### DISCUSSION TITLE
alternative RFC formats

### DISCUSSION THREAD

**Discussion by ghost** [ietf-tools/datatracker#4776] [Help / Questions] [answered]
Hi everyone.

I was accessing and reading this link here: [choosing-a-format-and-tools](https://authors.ietf.org/en/choosing-a-format-and-tools)

> **Note**
> "The following authoring formats are in use by the IETF community, though some by only a handful of people: RFCXML, Markdown, Other markup languages, Plain text, [Word processors(MS Word with RFC 5385 template and tool, MS Word with RFC Tool or xml2docx)](https://authors.ietf.org/drafting-in-word-processors)"

I wonder if you've all thought about using [jekyll](https://github.com/just-the-docs/just-the-docs), [hugo-book](https://github.com/hotosm/hugo-book) or [tiddlywiki](https://nicolas.petton.fr/tw/notebook.html) to document RFC.

*This is not a request for a resource, but a question that I am curious about. The reason for this question is that tiddlywiki works offline and has a lot of text formatting features. Also, it is a single html document. This can help in creating, maintaining, developing RFC files. But there are formats like "hugo-book" and "just-the-docs" that are visually interesting too.*

**Comment by JayDaley** [comment, marked as answer]
I think you may have misunderstood the process here.  RFCs begin life by people writing an Internet-Draft and there are no rules about what formats people can choose to author an Internet-Draft in, but it must be possible to convert that format into either RFCXML or the a highly structured plain text document.  The list you have quoted above is the list of formats that people choose to author Internet-Drafts in.  If you or anyone else wanted to use one of those tools you suggest then there is nothing to stop them.

**Reply by ghost** [reply]
Hi JayDaley, thank you for feedback.
